File: app/src/main/java/org/icspl/icsconnect/models/GrpConversationModel.java
package org.icspl.icsconnect.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GrpConversationModel {


    @SerializedName("Group_Messages")
    @Expose
    public List<GroupMessage> groupMessages = null;


    public class GroupMessage {

        @SerializedName("Srno")
        @Expose
        public Integer srno;
        @SerializedName("Group_id")
        @Expose
        public String groupId;
        @SerializedName("Q_Message")
        @Expose
        public String qMessage;
        @SerializedName("Q_attachment")
        @Expose
        public String qAttachment;
        @SerializedName("Q_date")
        @Expose
        public String qDate;
    }

}

File: app/src/main/java/org/icspl/icsconnect/models/GrpConversationModel.kt
package org.icspl.icsconnect.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class GrpConversationModel {


    @SerializedName("Group_Messages")
    @Expose
    var groupMessages: List<GroupMessage>? = null


    inner class GroupMessage {

        @SerializedName("Srno")
        @Expose
        var srno: Int? = null
        @SerializedName("Group_id")
        @Expose
        var groupId: String? = null
        @SerializedName("Q_Message")
        @Expose
        var qMessage: String? = null
        @SerializedName("Q_attachment")
        @Expose
        var qAttachment: String? = null
        @SerializedName("Q_date")
        @Expose
        var qDate: String? = null
    }

}
